docs/cristin.ttl

```diff
@@ -16,35 +16,55 @@
 :Published
     a               rdfs:Class ;
     rdfs:label      "PUBLISHED" ;
+    rdfs:label      "PUBLISERT" ;
     rdfs:subClassOf :PublicationStatus .
 
+:InPress
+    a               rdfs:Class ;
+    rdfs:label      "IN_PRESS" ;
+    rdfs:subClassOf :PublicationStatus .
+
+::TestPublicationStatus
+    a               rdfs:Class ;
+    rdfs:label      "TEST" ;
+    rdfs:subClassOf :PublicationStatus .
+
+
 :VARBEID_SPRAK_Class
     a            rdfs:Class ;
     rdfs:comment "The structure defining the VARBEID_SPRAK property." .
 
 :VARBEID_PERSON_Class
     a            rdfs:Class ;
     rdfs:comment """
-    The structure of the VARARBEID_PERSON entry. It contains information about the status of the contributor
-    at the time the Result was registered in Cristin.
+    The structure of the VARARBEID_PERSON entry. It contains user submitted information about the status of the contributor
+    at the time the Result was registered in Cristin. Some of the information is provided by the user manually in order to reflect the
+    user's name at the time the manuscript/publication was submitted to the publisher (e.g. Elsevier).
     """ .
 
 :VARBEID_PERSON_STED_Class
     a            rdfs:Class ;
     rdfs:comment """
     The structure of the VARBEID_PERSON_STED entry.
     It contains information about the status of a contributor's institution at the time the Result
-    was registered in Cristin. It is uncertain whether the contributor presently belongs to the same institution,
-    or if the institution status has changed.
+    was registered in Cristin. If the institution which the contributor belonged when the Cristin entry was created has now changed,
+    then this will be reflected in the entry. However the fact that the  contributor has moved to another institution is not reflected here.
+
+    Example:
+    Contributor C worked for for institution "InstA" when they registered publication P in Cristin.
+    "InstA" has merged with another institution and now is called "InstB".
+    Contributor C is working today for "InstC". The Cristin entry will indicate the fact that:
+    "When Contributor C was affiliated with InstB, they published P".
     """ .
 
 :VARBEID_PERSON_STED_ROLLE_Class
     a            rdfs:Class ;
     rdfs:comment """
      The structure of VARBEID_PERSON_STED_ROLLE. It contains information about the role of
-    the contributor in their institution at the time the Result was registered in Cristin. It is uncertain
-    whether the contributor presnetly holds the same role, or they belong to the same institution, or the
-    role still exists, or the institution's status has changed.
+    the contributor in this publication. One contributor can have two different contribution roles from different institutions.
```

Comment:
Not technically limited to two, and don't even technically have to be at different institutions.

Unless you have received opposing information from e.g. Jon, if I understand correctly, there are business rules in place that in practice limit the cases you'll see to two, but it's not a fixed, implemented feature of the data model.